runtime/src/lib.rs

```diff
@@ -585,6 +589,32 @@ impl additional_traits::DelegatedDispatchVerifier for Runtime {
 	}
 }
 
+/// Handles gas payment post contract execution (before deferring runtime calls) via CENNZX-Spot exchange.
+impl<T> pallet_contracts::GasHandler<T> for Runtime
+where
+	T: pallet_contracts::Trait + pallet_generic_asset::Trait + crml_cennzx_spot::Trait + Decode,
+{
+	fn empty_unused_gas(transactor: &T::AccountId, gas_meter: GasMeter<T>) {
+		let key = &b"cennzx-spot"[..];
```

Comment:
Could throw this into constants.rs and would prefer a name related to fee exchange / gas

Reply:
We will use something like this: `const GAS_FEE_EXCHANGE = "cennzx-spot"`

Reply:
can you change the actual key byte string to match the variable name also?